common/presenters/table/object-to-table-head.js

```diff
@@ -1,15 +1,45 @@
 const { pickBy } = require('lodash')
+const queryString = require('query-string')
 
 const i18n = require('../../../config/i18n')
 
+function buildUrl(head) {
+  const {
+    currentUrl: { pathname, query },
+    key,
+    currentSort,
+  } = head
+  return queryString.stringifyUrl({
+    url: pathname,
+    query: {
+      ...query,
+      sortBy: key,
+      sortDirection: currentSort,
+    },
+  })
+}
+
+function buildWrapper(head) {
+  const label = i18n.t(head.html)
+  const url = buildUrl(head)
+  let sort = 'none'
+  if (head.currentSort) {
+    sort = head.currentSort === 'asc' ? 'descending' : 'ascending'
```

Comment:
This logic seems the wrong way around?

Reply:
On second thought -- I actually reverse it twice. Before passing it, to build the url, and then again to set the arrow.
It should be enough to reverse it once at the end.